analyticConfigurations/computed_metric-route53.json

```diff
@@ -0,0 +1,19 @@
+{
+  "analyticConfiguration": {
+    "metrics": [
+      {
+        "match": "aws.route53.healthecheckstatus.min",
+        "properties": {
+          "expression": "data.min('aws.route53.healthecheckstatus.*')",
+          "fqn": "aws.route53.healthecheckstatus.min",
```

Comment:
@rclagett and @Grumblesmith: Are we still prefixing computed metrics with `netuitive`? Also, this new FQN matches your RegEx. I'm not sure if that will mess up the calculation.

Reply:
I've fixed a typo in aws.route53.healthcheckstatus.min and removed meta for this metric.